VERSION

```diff
@@ -1 +1 @@
-v1.7.3
```

Comment:
No newline in the VERSION string. You don't have to update it anymore. We now flash based on firmware hash. We should remove the version altogether.

Reply:
Fixed!